user-guide/Standard_Apps/PA/PA concepts/Activities.md

```diff
@@ -0,0 +1,41 @@
+---
+uid: PA_Activities
+---
+
+# Activities
+
+A process is a collection of granular activities interconnected with each other to support a main objective. Up until now, these activities did not exchange data, as each activity is responsible for performing a specific task and is not aware of the other activities composing the process.
+
+A DataMiner Object Model (DOM) instance named “process DOM instance”, however, will be used as a container of data, which the various activities within a process can retrieve or update. This way, the different activities *can* exchange data.
+
+## Supported Activities
+
+Process automation supports the following types of activities:
+
+- [**Script tasks**](xref:Creating_Activities#script-tasks): A script task is an Automation script with a specific signature. It can be used to perform any custom action. The custom logic is fully part of the script and can of course interface with any DataMiner feature/object.
+
+- [**User tasks**](xref:Creating_Activities#user-tasks): A user task is an activity that will wait for a user action before going to the next activity. Typically, a User Task app will be made available so that end user can access these User tasks and complete them.
```

Comment:
some corrections are still needed here (e.g. the end user" instead of "end user")